### DISCUSSION TITLE
Can't find 'retrieval' setting

### DISCUSSION THREAD

**Discussion by eschewary** [janhq/jan#2546] [Get Help]
I can't find any 'retrieval' setting. How can I get Jan to react to images? When I try to post an image, I see this:

![image](https://github.com/janhq/jan/assets/110819505/ee9bd318-9d93-48e6-af5b-cd82c404af98)

But I don't see any assistant settings anywhere in the program:

![image](https://github.com/janhq/jan/assets/110819505/af00c6e3-9e7c-4c5d-9c5b-1615047ba667)

It's also not listed under 'advanced settings'. I'm using the latest version available, (0.4.9):

![image](https://github.com/janhq/jan/assets/110819505/f2e53ed9-5797-4040-b73e-1b6266c14fa0)

Where is the retrieval setting?

**Comment by eschewary** [comment]
I found it, it's in the right panel under the parameter settings. Issue closed

**Reply by khanakia** [reply]
Can you send me the screenshot please i can't find it either

**Reply by p-slatts** [reply]
I am also struggling to find this, a screenshot would be very helpful

**Reply by khanakia** [reply]
Go to settings enable the Experimental Mode first
and then go the chat window and you will see it under the Tools

![Screenshot 2024-07-18 at 7 01 46 PM](https://github.com/user-attachments/assets/7c6221ae-3f5b-4907-bc62-650f439e36bc)

![Screenshot 2024-07-18 at 7 02 09 PM](https://github.com/user-attachments/assets/94a0f74d-2554-439c-86b8-59559f2381e7)

**Comment by octosteve** [comment]
Is that only available for remote models? I'm running it locally and not seeing this.
